generators/server/templates/src/main/java/package/config/SecurityConfiguration.java.ejs

```diff
@@ -20,80 +20,77 @@ package <%= packageName %>.config;
 import org.springframework.security.config.http.SessionCreationPolicy;
-<%_ } _%>
-<%_ if (authenticationType !== 'oauth2' && !skipUserManagement) { _%>
+    <%_ } _%>
+    <%_ if (authenticationType !== 'oauth2' && !skipUserManagement) { _%>
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
-<%_ } _%>
-<%_ if (authenticationType === 'oauth2') { _%>
-import org.springframework.core.convert.converter.Converter;
-import org.springframework.security.authentication.AbstractAuthenticationToken;
+    <%_ } _%>
+    <%_ if (authenticationType === 'oauth2') { _%>
 import <%= packageName %>.security.oauth2.AudienceValidator;
+import <%=packageName%>.security.SecurityUtils;
 import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator;
 import org.springframework.security.oauth2.core.OAuth2TokenValidator;
 import org.springframework.security.oauth2.jwt.*;
-import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationConverter;
-    <%_ if (applicationType === 'gateway') { _%>
+        <%_ if (applicationType === 'gateway') { _%>
 import <%=packageName%>.security.oauth2.AuthorizationHeaderFilter;
 import <%=packageName%>.security.oauth2.AuthorizationHeaderUtil;
-    <%_ } _%>
+        <%_ } _%>
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.security.core.GrantedAuthority;
-import org.springframework.security.core.authority.SimpleGrantedAuthority;
-<%_ if (applicationType !== 'microservice') { _%>
+        <%_ if (applicationType !== 'microservice') { _%>
 import org.springframework.security.core.authority.mapping.GrantedAuthoritiesMapper;
 import org.springframework.security.oauth2.core.oidc.user.OidcUserAuthority;
-<%_ } _%>
+        <%_ } _%>
 import java.util.*;
-import java.util.stream.Collectors;
-<%_ } _%>
-<%_ if (authenticationType === 'session') { _%>
-    <%_ if (!skipUserManagement) { _%>
-import org.springframework.security.web.authentication.RememberMeServices;
     <%_ } _%>
+    <%_ if (authenticationType === 'session') { _%>
+        <%_ if (!skipUserManagement) { _%>
+import org.springframework.security.web.authentication.RememberMeServices;
+        <%_ } _%>
 import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
 import org.springframework.security.web.csrf.CsrfFilter;
-<%_ } _%>
-<%_ if (authenticationType === 'oauth2' && applicationType !== 'microservice') { _%>
+    <%_ } _%>
+    <%_ if (authenticationType === 'oauth2' && applicationType !== 'microservice') { _%>
 import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
 import org.springframework.security.web.csrf.CsrfFilter;
-<%_ } _%>
-<%_ if (authenticationType === 'jwt' && applicationType !== 'microservice') { _%>
+import <%=packageName%>.security.oauth2.JwtAuthorityExtractor;
```

Comment:
@Tcharl : should we include `JwtAuthorityExtractor` for microservice ? I think yes, otherwise, OAuth2+Microservice doesn't compile